convert/pyproject.toml

```diff
@@ -28,7 +28,7 @@ isort = "^5.9.1"
 mypy = "^0.910"
 pylint = "^2.8.1"
 black = "^22.3"
-datamodel-code-generator = "^0.17.1"
+# datamodel-code-generator = "^0.17.1"
```

Comment:
Could you clarify this commented dependency?

Reply:
Still concerning my attention: why would we remove it? And why would we need it in future?

Reply:
Done